cloverage/project.clj

```diff
@@ -16,11 +17,12 @@
     :scm :git ; Because we're not in the top-level directory, so it doesn't auto-detect
     :deploy-via :clojars
   }
-  :dependencies [[org.clojure/clojure "1.4.0"]
-                 [org.clojure/tools.cli "0.2.2"]
-                 [org.clojure/tools.logging "0.2.3"]
-                 [org.clojure/data.xml "0.0.7"]
-                 [bultitude "0.2.0"]
-                 [riddley "0.1.4"]
-                 [slingshot "0.10.3"]
-                 [cheshire "5.3.1"]])
+  :dependencies [[org.clojure/clojure "1.6.0"]
```

Comment:
Primarily that I'm assuming it may exclude some people that would otherwise use the library. This may well be wrong, but I still don't want to upgrade unless there's a good reason, because not upgrading also avoids having to even check if that's right or not, and also results in a smaller set of changes for an individual PR